client/js/gdpr/chart/index.js

```diff
@@ -0,0 +1,345 @@
+import {SVG_NS} from '../../util.js';
```

Comment:
I think this file isn't supposed to show up for another PR or so :)

Reply:
No, it's there